app/messages.js

```diff
@@ -1,12 +1,18 @@
-define(['jquery'], function ($) {
+define(['jquery', 'app/timer'], function ($, timer) {
   /* Function show for displaying normal, danger and warning messages.
   * The message is included as html and a class with the corresponding type is
   * added. Within the switch statement the icon is added.
   */
 
   return {
-    show: function(message, type) {
-      $("#show .message").html(message);
+    show: function(message, type, countdown) {
+      countdown /= 1000;
```

Comment:
Magic number. You don't need to add it to the config, if that doesn't make sense, you can also add a variable with the value 1000, and use that for the division.